stko/calculators/extractors/orca_extractor.py

```diff
@@ -0,0 +1,114 @@
+"""
+Orca Extractor
+=============
+
+#. :class:`.OrcaExtractor`
+
+Class to extract properties from Orca output.
+
+"""
+
+import re
+
+
+class OrcaExtractor:
+    """
+    Extracts properties from Orca 4.2 output files.
+
+    Limited to final single point energy for now.
+
+    Attributes
+    ----------
+    output_file : :class:`str`
+        Output file to extract properties from.
+
+    output_lines : :class:`list` : :class:`str`
+        :class:`list` of all lines in as :class:`str` in the output
+        file.
+
+    total_energy : :class:`float`
+        The total energy in the :attr:`output_file` as
+        :class:`float`. The energy is in units of a.u..
+
+    """
+
+    def __init__(self, output_file):
+        """
+        Initializes :class:`OrcaExtractor`
+
+        Parameters
+        ----------
+        output_file : :class:`str`
+            Output file to extract properties from.
+
+        """
+
+        self.output_file = output_file
+        # Explictly set encoding to UTF-8 because default encoding on
+        # Windows will fail to read the file otherwise.
+        with open(self.output_file, 'r', encoding='UTF-8') as f:
+            self.output_lines = f.readlines()
+
+        self._extract_values()
```

Comment:
Agreed. Changed.

Reply:
I actually do agree, but will not make this change (unless you have a good reason for me to do it) because a proper implementation would break backwards compatibility and it is not worth it. @stevenkbennett